### ISSUE TITLE
The app shows the beacons for a short time, then crashes and after that "no data found" is shown

### ISSUE THREAD

**Issue by Piiit** [noi-techpark/it.bz.beacon.admin.android#39]
We tested version 0.5.1 and were able to see the full list of beacons and nearby beacons as well. Then after a restart or crash (Stefanos app crashed) no data was shown anymore. 

Before we also were able to do a trusted update, but only once...

**Comment by Piiit** [comment]
@noctho FYI

**Comment by Piiit** [comment]
We found out that the API key was wrong, but that should be shown as error... it should not crash, but show the list of beacons because INFO should be accessible for all of us, with a hint that the kontakt.io access was not possible and that therefore the beacon cannot be configured. Ideally, with a automated notification to our issue tracker if feasible

**Comment by noctho** [comment]
Yes, for this problem we could add a devide status that the API key is wrong or missing.

If the data on the device is complete, but the API key is wrong and the user tries to configure the beacon, they will see an error message that the API key is wrong and therefore they cannot configure the beacon.

**Comment by noctho** [comment]
> The app shows the beacons for a short time, then crashes and after that "no data found" is shown

I assume  that some beacons are already stored in your local device cache and then it refreshed your data the beacons are gone. Otherwise it is not possible that it shows some beacons. There was implemented a mechanism that ignores all beacon with incomplete data to avoid that the app crashes inside the details view. I have now changed it so that the beacons are also shown if the data is incomplete and it avoids that the app crashes inside the details view.
